src/System.Text.Json/Directory.Build.props

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\Directory.Build.props" />
+  <PropertyGroup>
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
```

Comment:
Is this the appropriate assembly version? cc @ericstj

Reply:
We usually start with 4.0.0.0 from as far as I know.